src/main/webapp/resources/js/components/Tables/AgGridLayout.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
```

Comment:
Add file comments

Reply:
Added in 82c9b53375c0ed3ac9654f10b56961948f76c406